### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by casals**
### Why:

This PR refers to issue #206 .

### What's being changed:

Added a small paragraph about the existence/use of PR templates when submitting a PR, right below the similar paragraph related to solving an issue.

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ As you're using the GitHub Docs, you may find something in an article that you'd
 
 If you've found a problem, you can open an issue using a [template](https://github.com/github/docs/issues/new/choose).
 
+#### Solve an issue
+
+If you have a solution to one of the open issues, you will need to fork the repository and submit a PR using a [template](https://github.com/github/docs/blob/main/CONTRIBUTING.md#pull-request-template). For more details about this process, please check out [Getting Started with Contributing](/CONTRIBUTING.md).
```
Comment (janiceilene) [inline README.md:31]:
```suggestion
If you have a solution to one of the open issues, you will need to fork the repository and submit a PR using the [template](https://github.com/github/docs/blob/main/CONTRIBUTING.md#pull-request-template) that is visible automatically in the pull request body. For more details about this process, please check out [Getting Started with Contributing](/CONTRIBUTING.md). 
```
What do you think about this @casals? Do you think that'll help people to understand that the template they need to fill out is the information that shows up automatically when they open a pull request?

Comment (casals) [reply README.md]: It's great, thanks! Already committed.

**Review by janiceilene — APPROVED** [review]
Thanks for this update @casals I'll merge it as soon as everything's green 💚

**Comment by janiceilene** [comment]
If you haven't already, you can add yourself to [the list of contributors](https://github.com/github/docs#contributors-) by creating a new comment in this PR using [these instructions](https://allcontributors.org/docs/en/bot/usage#commands). Thanks again! :sparkles:

**Pull request merged**